crates/anvil/src/eth/api.rs

```diff
@@ -2436,7 +2459,7 @@ impl EthApi {
                 m.chain_id = chain_id;
                 m.gas_limit = gas_limit;
                 if gas_price.is_none() {
-                    m.gas_price = self.backend.gas_price();
+                    m.gas_price = self.gas_price().unwrap().to::<u128>();
```

Comment:
We can make `fn gas_price() -> u128`, removing the result and using u128 directly to avoid this

Reply:
I have refactored this, though I hardly doubt this adjustment